src/transformers/models/deformable_detr/modeling_deformable_detr.py

```diff
@@ -141,23 +161,6 @@ def backward(context, grad_output):
         return grad_value, None, None, grad_sampling_loc, grad_attn_weight, None
 
 
-if is_scipy_available():
```

Comment:
Moves these above the MultiScaleDeformableAttentionFunction definition - better matching library patterns